src/sagemaker_training/process.py

```diff
@@ -39,6 +39,22 @@
 _DEFAULT_BUF_SIZE = 1024 * 64
 
 
+def get_debugger_exception_classes():
+    """Set exception classes"""
+    exception_classes = []
+    try:
+        from smdebug import exceptions
+
+        # list of exceptions debugger wants training toolkit to catch and log
+        exception_classes += [ex for ex in dir(exceptions) if isclass(getattr(exceptions, ex))]
+    except ImportError:
+        logger.info("No exception classes found in smdebugger")
```

Comment:
Let's rename this to `sagemaker debugger`

Reply:
What happens when the user turns off sagemaker debugger from the estimator call ? Do we still capture exceptions from the debugger package according to this logic ? Can we differentiate between debugger disabled vs no exception classes found in debugger?